src/managing-workflow/platform-monitoring.md

```diff
@@ -1,38 +1,40 @@
 # Platform Monitoring
 
 ## Description
-
-With the release of Deis Workflow, we now include a monitoring stack for introspection on a running Kubernetes cluster. The stack includes 4 components:
-
+With the release of Workflow Beta4 we now include a monitoring stack for introspection on a running Kubernetes cluster. The stack includes 3 components:
```

Comment:
Does talking about the betas even make sense anymore? Could this just be reduced to "Workflow includes..."